.github/workflows/build-docs.yml

```diff
@@ -0,0 +1,44 @@
+# This workflow installs the package and builds the docs
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: build the docs
+
+on:
+  push:
+    branches: [ 'main' ]
+  pull_request:
+    branches: [ '**' ]
+
+jobs:
+  pytest:
+    strategy:
+      matrix:
+        os:
+        - ubuntu-latest
+        python-version:
+        - '3.8'
+
+      fail-fast: false
+
+
+    runs-on: ${{ matrix.os }}
+    name: ${{ matrix.os }} py${{ matrix.python-version }}
+  
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+
+    - name: Install dependencies and package for building the docs
+      run: |
+        python -m pip install --upgrade pip
+        pip install -e .[doc]
+
+    - name: Build the docs
+      run: |
```

Comment:
This can be a one-liner:
https://github.com/iiasa/ixmp/blob/5c4a78c/.github/workflows/pytest.yml#L148-L149

Reply:
done, thanks - learned something!